api/src/pcapi/core/bookings/constants.py

```diff
@@ -11,6 +13,12 @@
 AUTO_USE_AFTER_EVENT_TIME_DELAY = datetime.timedelta(hours=48)
 REDIS_EXTERNAL_BOOKINGS_NAME = "api:external_bookings:barcodes"
 EXTERNAL_BOOKINGS_MINIMUM_ITEM_AGE_IN_QUEUE = 60
+FREE_OFFER_CATEGORIES_TO_ARCHIVE = [
```

Comment:
```suggestion
FREE_OFFER_SUBCATEGORIES_TO_ARCHIVE = [
```
éviterait une confusion entre catégories et sous-catégories